src/test/bundle_test.ts

```diff
@@ -383,16 +365,45 @@ suite('BuildBundler', () => {
     assert.include(bundledHtml, '.simply-red', 'simple-style.css');
   });
 
-  test('bundler does not output imports which have been bundled', async() => {
+  test(
+      'bundler does not output html imports which have been bundled',
+      async() => {
+        await setupTest({
+          entrypoint: 'entrypoint-only.html',
+          sources: ['framework.html', 'entrypoint-only.html'],
```

Comment:
I actually didn't change this test at all, just its name, to be similar to the new test for 'bundler does not output inlined scripts or styles'.  The diff didn't make that obvious.  Fixing that.

I went ahead and removed all other `sources:` entries from the configs, because they were all irrelevant as far as the bundler was concerned.  They really only effected the BuildAnalyzer, none of that mattered to the bundle tests.

Changes pushed.